LICENSE

```diff
@@ -1,14 +1,30 @@
-Copyright (c) 2005-2016, Regents of the University of California, the University of North Carolina at Chapel Hill, and the Data Intensive Cyberinfrastructure Foundation
-All rights reserved.
+BSD 3-Clause License
 
-Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+Copyright (c) 2005, Regents of the University of California
+Copyright (c) 2008, The University of North Carolina at Chapel Hill
```

Comment:
let's flip these so unc on top - then i think we're good

Reply:
Done.